archive/generic-core-terms.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Security is core to our values, and we value the input of hackers acting in good-faith to help us maintain a high standard for the security and privacy for our users. This includes encouraging responsible vulnerability research and disclosure. This policy sets out our definition of good-faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
+
+# Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Work with you to understand and validate your report, including a timely initial response to the submission;
+- Work to remediate discovered vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to report a unique vulnerability, and your report triggers a code or configuration change.
+
+# Scope
+
```

Comment:
Suggested insertion: 

The scope of this policy includes only assets owned by the party participating in this policy, specifically:
[...]

Any systems not listed above are out-of-scope for security testing under this policy. Any discovered or suspected vulnerabilities in out of scope systems should be reported to the appropriate vendor or applicable authority. 

Reply:
Regarding this, we were actually looking at adjusting the language for our own purposes (for VDP, which is designed to be open scope) to something like:

> This policy applies to any digital assets owned, operated, or maintained by $COMPANY for which $COMPANY can legally authorize the testing of, including instances of the $COMPANY product which you have explicit permission to test.

Reply:
@harleygeiger - Would @dantrauner's language work here? The only challenge I see is the tendency for organizations not to know to 100% which assets on the internet are theirs

Reply:
I would lean towards keeping terms general for a VDP, since it may not always be possible for an organization to enumerate their assets. 

Perhaps:

The scope of this policy includes only assets owned, oper by the party participating in this policy, specifically:

This policy applies to any assets owned, operated, or maintained by [organization], including the following:

...

If you are unsure whether a system belongs to us, please reach out before testing. Any discovered or suspected vulnerabilities in out of scope systems should be reported to the appropriate vendor or applicable authority. 

Reply:
> @HarleyGeiger - Would @dantrauner's language work here? The only challenge I see is the tendency for organizations not to know to 100% which assets on the internet are theirs

Maybe combine them:

This policy applies only to any digital assets owned, operated, or maintained by $COMPANY for which $COMPANY can legally authorize the testing of. Any assets not listed above are out-of-scope for security testing under this policy. Vulnerabilities discovered or suspected in out-of-scope systems should be reported to the appropriate vendor or applicable authority. 